### PR TITLE
Offset update

### PR DESCRIPTION
**Pull request by 0xfffffff0075e2040**
Updated offset for macOS 10.14.4.

### CODE REVIEW

File: iMessage/dumpMessages.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ function pm(p) {
 }
 
 // Offset from macOS 10.14.3
```
Comment (Ahmed28hamouda) [inline iMessage/dumpMessages.js:60]:
```suggestion
// Offset from macOS 10.14.3
```

**Comment by miouvhhuhvhv** [comment]
How to change offset vue

**Comment by miouvhhuhvhv** [comment]
Tell me the offset value for macos 11

File: iMessage/dumpMessages.js (hunk 1 above)
Comment (miouvhhuhvhv) [reply iMessage/dumpMessages.js:60]: Bro do you know how change offset value. Coul you tell me

Comment (miouvhhuhvhv) [reply iMessage/dumpMessages.js:60]: I try to contact evey developer no one reply me. I need help

**Comment by tepejof80** [comment]
i have a same issue. tell me please offset value. i don't know offset value

**Pull request closed without merging**